plenum/recorder/replayable_node.py

```diff
@@ -87,7 +87,7 @@ def create_replicas(self, config=None):
             return _TestReplicas(self, self.monitor, config)
 
         def adjustReplicas(self):
```

Comment:
Why does this method violate the base class contract?

Reply:
Fixed it.